### DISCUSSION TITLE
automatically delete desktop icon after installation

### DISCUSSION THREAD

**Discussion by lsol4401** [Devolutions/UniGetUI#2741] [Q&A]
Hello everyone. Is there a way to automatically delete the desktop icons after installation? Thanks.

**Comment by marticliment** [comment]
Not yet

**Reply by lsol4401** [reply]
Thanks a lot for answering. 
Perhaps this could be added as a new function at some point. I think that would help a lot, especially in assisted family use.

**Reply by marticliment** [reply]
I will investigate
